client/src/styles.tsx

```diff
@@ -3,11 +3,11 @@ import styled, { createGlobalStyle } from 'styled-components';
 // Global styles
 
 export const GeneralSectionContainer = styled.div`
-  height: 100vh;
+  min-height: 100vh;
   width: 100%;
   display: flex;
   flex-direction: column;
-  justify-content: center;
+  justify-content: flex;
```

Comment:
Everything looks good. Just double-check this justify-content: flex. I don't think flex is an option for justify-content.